docker/ubuntu-full/Dockerfile

```diff
@@ -563,7 +563,9 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     && apt-get update \
     && apt-get install -y -V libarrow${ARROW_SOVERSION} \
     && apt-get install -y -V libparquet${ARROW_SOVERSION} \
-    && apt-get install -y -V libarrow-dataset${ARROW_SOVERSION}
+    && apt-get install -y -V libarrow-dataset${ARROW_SOVERSION} \
+    && apt-get clean \
```

Comment:
This clears the packages/lists from Docker cache volume mounted in lines 524-525.